### ISSUE TITLE
jenkins/GH hooks are not working correctly anymore

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#7134]
I noticed that Jenkins execute test job now opens a new issue for every message, instead of adding comments to initial issue. I guess something was changed ?
![Screenshot from 2022-03-16 15-13-46](https://user-images.githubusercontent.com/1837785/158610042-ef71bb99-113c-431b-8cce-5a7029075869.png)

**Comment by mapellidario** [comment]
Something is off.. I looked at our own script. It seems to send the data to github via commands like

```bash
$WORKSPACE/cms-bot/create-gh-issue.py -r dmwm/CRABServer -t "$issueTitle" -R message_taskSubmitted
```

I tried to replicate it via 
- cloning https://github.com/cms-sw/cms-bot/, 
- install the dependency with `python -m pip install PyGithub`
- creating a personal access token (scope `repo:status`, `repo_public`), saving it to `~/.github-token`
- executing: 

    ````plaintext
    user@1adf2bbcdf77:~/mnt$ python create-gh-issue.py -r mapellidario/CRABServer -t "[test] open GH issue from with cmsbot" -m     "[test] message of the issue opened via cms-sw/cms-bot/create-gh-issue.py"
    Authenticating to Github and connecting to repo
    Authentication succeeeded
    Creating issue request
    user@1adf2bbcdf77:~/mnt$ python create-gh-issue.py -r mapellidario/CRABServer -t "[test] open GH issue from with cmsbot" -m     "[test] message of the issue opened via cms-sw/cms-bot/create-gh-issue.py, second message"
    Authenticating to Github and connecting to repo
    Authentication succeeeded
    Updaing comment
    user@1adf2bbcdf77:~/mnt$ 
    ````
- and it correctly creates the issue the first time, while added a comment the second time https://github.com/mapellidario/CRABServer/issues/8 (provided that the title is the same, as it appears from reading the `create-gh-issue.py` code.)

In conclusion, the cause of the error we saw the other is not clear and requires some more time...

**Comment by belforte** [comment]
everything seems to work again now that GitHub fixed their problems
